app/screens/orders/index.js

```diff
@@ -1,13 +1,139 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+// import React from 'react';
+import * as React from 'react';
+import { Text, View, ScrollView, StyleSheet }  from 'react-native';
+import styles from './styles';
 
-const Orders = () => {
+
+export default Orders = () => {
 
   return(
-  <View>
-    <Text>Orders</Text>
-  </View>
+    <ScrollView>
+            <View style={{flex:1}}>
+
+            <View>
+            <Text style={styles.orderHistoryTitle}>Order History</Text>
```

Comment:
I think you can still improve on this whole screen by using a Flatlist that renders the items that you have.

You've already shown me how you did it before. Just implement it here.

Reply:
Noted po! Thank you 😃